roles/serviceassurance/tasks/component_elasticsearch.yml

```diff
@@ -15,9 +15,9 @@
               memory: 4Gi
             requests:
               memory: 1Gi
-        redundancyPolicy: {{ "FullRedundancy" if haEnabled else "ZeroRedundancy" }}
+        redundancyPolicy: {{ "FullRedundancy" if highAvailabilityEnabled else "ZeroRedundancy" }}
         nodes:
-          - nodeCount: {{ esNodeCount if haEnabled else "1" }}
+          - nodeCount: {{ elasticSearchNodeCount if highAvailabilityEnabled else "1" }}
```

Comment:
`elasticsearch_node_count` and `high_availability_enabled`

Reply:
Instead of peppering all of our files with ` if high_availability_enabled else <default> ` is it possible to move this to one central place? I haven't looked into how to do this in our ansible yet, but pseudo-code I'm basically looking for this:
```
# Defaults for noHA
elasticsearch_node_count = 1
qdr_node_count = 1

# Defaults for HA
if high_availability_enabled then {
 elasticsearch_node_count = 3
 qdr_node_count = 2
}
```

Reply:
Sounds like you want to have a `set_fact` in the `roles/serviceassurance/tasks/main.yml` that sets the variables conditionally. Basically same thing we're doing here in the manifest configuration, but just have that manifest reference the variables, and move their dynamic parameter sets in a single location.

Reply:
Something like this, then: https://github.com/redhat-service-assurance/service-assurance-operator/pull/26